### PR TITLE
Refactor timeout

### PR DESCRIPTION
**Pull request by slackve**

### CODE REVIEW

**Comment by alvin-reyes** [comment]
Please fix the conflicts.

File: sdk/subscribe_block.go [hunk 1]
```diff
@@ -114,8 +113,9 @@ func (c *SubscribeService) getClient(add string) *ClientWebsocket {
 		return c.client
 	} else {
 		client, err := NewConnectWs(c.client.config.BaseURL.String(), *c.client.duration)
+
 		if err != nil {
-			fmt.Println(err)
+			panic(err)
```
Comment (ruslanBik4) [inline sdk/subscribe_block.go]: don't use panic - use error return

Comment (slackve) [reply sdk/subscribe_block.go]: fixed in https://github.com/proximax-storage/nem2-sdk-go/commit/e7f3dc739c27ae59cedd2846cc881ea4e1167569

**Comment by alvin-reyes** [comment]
There are conflicts.

**Comment by alvin-reyes** [comment]
still need to resolve conflicts before merging.

**Pull request closed without merging**